src/InsertionsClient.Console/ConsoleApp/Program.cs

```diff
@@ -137,14 +144,20 @@ private static int Main(string[] args)
             {
                 ignoredPackages = InsertionConstants.DefaultDevUxTeamPackages;
             }
+
+            if(!string.IsNullOrWhiteSpace(BuildFilterString) && !InputLoading.LoadBuildFilter(BuildFilterString, out buildFilter))
```

Comment:
seeing this line, makes me more strongly recommend renaming `LoadBuildFilter` => `TryLoadBuildFilter`